.circleci/config.yml

```diff
@@ -9,29 +9,37 @@ references:
   restore-dependencies-cache: &restore-dependencies-cache
     restore_cache:
       keys:
-        - deps-py<< parameters.version >>-{{ checksum "poetry.lock" }}
+        - deps-py<< parameters.python-version >>-django<< parameters.django-version >>-{{ checksum "poetry.lock" }}
   install-dependencies: &install-dependencies
     run:
       name: Install Dependencies
       command: |
         source $HOME/.poetry/env
         poetry install
+        poetry run pip install "django>=<< parameters.django-version >>"
   save-dependencies-cache: &save-dependencies-cache
     save_cache:
-      key: deps-py<< parameters.version >>-{{ checksum "poetry.lock" }}
+      key: deps-py<< parameters.python-version >>-django<< parameters.django-version >>-{{ checksum "poetry.lock" }}
       paths:
         - /home/circleci/.cache/pypoetry/virtualenvs
   parametrised-python-executor: &parametrised-python-executor
     parameters:
-      version:
+      python-version:
+        type: string
+      django-version:
         type: string
     executor:
       name: python
-      version: << parameters.version >>
-  python-version-matrix: &python-version-matrix
+      version: << parameters.python-version >>
+  version-matrix: &version-matrix
     matrix:
       parameters:
-        version: ["3.5.5", "3.6", "3.7", "3.8"]
+        python-version: ["3.5.5", "3.6", "3.7", "3.8"]
+        django-version: ["1.11.27", "2.2", "3.1"]
```

Comment:
Since Django 1.11 is out of support, need it be included?

Reply:
Now covered by 6af0cc1